site/app/templates/course/EditCourseMaterialsForm.twig

```diff
@@ -17,21 +17,25 @@
             </label>
             <br>
             <span id = "show-some-section-selection-edit" style="display: none">
```

Comment:
You're getting web validator errors because in HTML a `span` is considered an inline element but a `table` is a block element. If you change this from a `span` to a `div` it should work fine and stop complaining

See more information here
https://developer.mozilla.org/en-US/docs/Web/HTML/Inline_elements

Reply:
Thanks!